### PR TITLE
Create controls and fix old ones

### PR DESCRIPTION
**Pull request by xadaemon**
Added a property to all classes derived from checkbox and add a PassWord class
Examples
```XML
<PassWord PassChar="*"></PassWord>
<CheckBox CheckedChar="■"></CheckBox>
```
Signed-off-by: Matheus Silva <matheus@codealkemy.co>

### CODE REVIEW

File: ConsoleFramework/Controls/PassWord.cs [hunk 1]
```diff
@@ -0,0 +1,288 @@
+﻿using System;
+using System.Collections.Generic;
+using ConsoleFramework.Core;
+using ConsoleFramework.Events;
+using ConsoleFramework.Native;
+using ConsoleFramework.Rendering;
+
+namespace ConsoleFramework.Controls
+{
+    public class PasswordBox : Control
```
Comment (igor-kostromin) [inline ConsoleFramework/Controls/PassWord.cs:10]: mb rename the file from `PassWord.cs` to `PasswordBox.cs` for consistency ?

File: readme.md [hunk 2]
```diff
@@ -13,12 +15,14 @@ Features
 - A lot of controls available (including Grid, ScrollViewer, ListBox, ComboBox)
 - Routed events system (compatible with WPF)
 - Windows, Mac OS X and any Linux (32-bit or 64-bit) support
+- Currently with 23 controlls more to be added
 
-![](http://gyazo.com/81e1ae92cfba8c7a1c2a98da7da75ad7.png)
+![]()
```
Comment (igor-kostromin) [inline readme.md:20]: I can agree to replace my screenshot with something another, but not to just remove them :)

File: ConsoleFramework/ConsoleFramework.csproj [hunk 3]
```diff
@@ -67,10 +67,12 @@
     <Compile Include="Controls\ListBox.cs" />
     <Compile Include="Controls\Menu.cs" />
     <Compile Include="Controls\MessageBox.cs" />
+    <Compile Include="Controls\PassWord.cs" />
     <Compile Include="Controls\ProgressBar.cs" />
     <Compile Include="Controls\RadioButton.cs" />
     <Compile Include="Controls\ScrollViewer.cs" />
     <Compile Include="Controls\TabControl.cs" />
+    <Compile Include="Controls\TextArea.cs" />
```
Comment (igor-kostromin) [inline ConsoleFramework/ConsoleFramework.csproj:75]: I dont see this file in PR.. is it committed ?

File: ConsoleFramework/ConsoleApplication.cs [hunk 4]
```diff
@@ -17,6 +17,36 @@
 
 namespace ConsoleFramework
 {
+
+    /// <summary>
+    /// Gets a pre-started window
+    /// </summary>
+    public static class EasyAPI
```
Comment (igor-kostromin) [inline ConsoleFramework/ConsoleApplication.cs:24]: What is the usecase for these classes ? I don't see any example code

**Comment by xadaemon** [comment]
Needs fixing of these issues they went unseen

**Pull request closed without merging**